File: BarkOrBolt-iOS/BarkOrBolt-iOS/dogObject.swift
//
//  dogObject.swift
//  BarkOrBolt-iOS
//
//  Created by Khuram Chaudhry on 4/11/17.
//  Copyright © 2017 Silicon Empire LLC. All rights reserved.
//  Data Model for the dogUser.

import UIKit

class dogObject {
    
    var dogImage: UIImage?
    var name: String
    var email: String
    var dateJoined: Date
    var breed: String
    var gender: String
    var ownerName: String
    
    init(dogImage: UIImage?, name: String, email: String, dateJoined: Date, breed: String, gender: String,
         ownerName: String) {
        self.dogImage = dogImage
        self.name = name
        self.email = email
        self.dateJoined = dateJoined
        self.breed = breed
        self.gender = gender
        self.ownerName = ownerName
    }
    
    func getDogImage() -> UIImage {
        return self.dogImage!
    }
    
    func getName() -> String {
        return self.name
    }
    
    func getEmail() -> String {
        return self.email
    }
    
    func getDateJoined() -> Date {
        return self.dateJoined
    }
    
    func getBreed() -> String {
        return self.breed
    }
    
    func getGender() -> String {
        return self.gender
    }
    
    func getOwnerName() -> String {
        return self.ownerName
    }
    
}

File: BarkOrBolt-Android/app/src/main/java/siliconempirellc/barkorbolt_android/Models/dogObject.java
package siliconempirellc.barkorbolt_android.Models;

import android.graphics.drawable.Drawable;

import java.io.File;
import java.util.Date;

/**
 * Created by khuramchaudhry on 4/9/17.
 * Data Model for the dogUser.
 */

public class dogObject {

    Drawable dogImage;
    String name;
    String email;
    Date dateJoined;
    String breed;
    String gender;
    String ownerName;

    public dogObject(Drawable dogImage, String name, String email, Date dateJoined, String breed,
                     String gender, String ownerName) {
        this.dogImage = dogImage;
        this.name = name;
        this.email = email;
        this.dateJoined = dateJoined;
        this.breed = breed;
        this.gender = gender;
        this.ownerName = ownerName;
    }

    public Drawable getDogImage() {
        return this.dogImage;
    }

    public String getName() {
        return this.name;
    }

    public String getEmail() {
        return this.email;
    }

    public Date getDateJoined() {
        return this.dateJoined;
    }

    public String getBreed() {
        return this.breed;
    }

    public String getGender() {
        return this.gender;
    }

    public String getOwnerName() {
        return this.ownerName;
    }

}
